### PR TITLE
Support Java 11

### PR DESCRIPTION
**Pull request by Camerooooon**
This way people can start the launcher in an outdated java version but QME won't start if the version isn't 15.

### CODE REVIEW

File: src/main/java/org/qme/installer/Installer.java [hunk 1]
```diff
@@ -42,12 +42,16 @@
...
-            case "linux" -> new LinuxInstaller();
-            case "mac", "mac os x" -> new MacInstaller();
-            default -> null;
-        };
+
+        if (os.equalsIgnoreCase("windows 10")) {
+            return new WindowsInstaller();
+        } else if (os.equalsIgnoreCase("linux")) {
+            return new LinuxInstaller();
+        } else if (os.equalsIgnoreCase("mac")) {
```
Comment (adamhutchings) [inline src/main/java/org/qme/installer/Installer.java:50]: Needs to have something for `"mac os x"` as well.

Comment (JakeRoggenbuck) [inline src/main/java/org/qme/installer/Installer.java]: You might be missing a "mac os x"

Comment (JakeRoggenbuck) [reply src/main/java/org/qme/installer/Installer.java]: same as previous comment

**Review by JakeRoggenbuck — APPROVED** [review]
Looks great!

**Pull request merged**